hardware/hcl.md

```diff
@@ -13,33 +13,34 @@ redirect_from:
 Hardware Compatibility List (HCL) for All Qubes OS Releases
 ===========================================================
 
-The [HCL](/hcl) is a compilation of reports generated and submitted by users across various Qubes versions.
+The [HCL](/hcl) is a compilation of reports generated and submitted by users across various Qubes versions about their hardware's compatibility with Qubes.
+
  **Note:**
  Except in the case of developer-reported entries, the Qubes team has not independently verified the accuracy of these reports.
  Please first consult the data sheets (CPU, chipset, motherboard) prior to buying new hardware for Qubes.
- Meet the [System Requirements](/doc/system-requirements/) and search particular for support of:
+ Make sure it meets the [System Requirements](/doc/system-requirements/) and search in particular for support of:
 
 -   HVM ("AMD virtualization (AMD-V)", "Intel virtualization (VT-x)", "VIA virtualization (VIA VT)")
 -   IOMMU ("AMD I/O Virtualization Technology (AMD-Vi)", "Intel Virtualization Technology for Directed I/O (VT-d)")
 -   TPM ("Trusted Platform Module (TPM)" connected to a "20-pin TPM header" on motherboards.)
 
-Test the hardware yourself, if possible.
- If using the list to make a purchasing decision, we recommend that you choose hardware with:
+If using the list to make a purchasing decision, we recommend that you choose hardware with:
 
 -   the best achievable Qubes security level (green columns in HVM, IOMMU, TPM)
 -   and general machine compatibility (green columns in Qubes version, dom0 kernel, remarks).
 
 Generating and Submitting New Reports
 -------------------------------------
 
-In order to generate a HCL report in Qubes, simply open a terminal in dom0 (KDE: start-menu -\> System Tools -\> Konsole or Terminal Emulator)
- and run `qubes-hcl-report <vm-name>`, where `<vm-name>` is the name of the VM to which the generated HCL files will be saved.
- (Note: If you are working with a new Qubes installation, you may need to update your system in order to download this script.)
+In order to generate a an HCL report in Qubes, simply open a terminal in dom0 (KDE: start-menu -\> System Tools -\> Konsole or Terminal Emulator)
```

Comment:
The acronym is pronounced "Aych See Ell". The letter in the acronym is H but it's the sound that counts.